scripts/plot_benchmark.py

```diff
@@ -52,15 +57,23 @@ def _plot_scenario_comparison(
     scenario: str,
     cyear: int,
     model_col: str,
-    rfc_col: str,
+    rfc_col: list[str],
     source_unit: str,
 ):
     fig, ax = plt.subplots(figsize=(12, 8))
 
     table_title = table.replace("_", " ").title()
-    df.set_index("carrier")[[model_col, rfc_col]].plot.bar(
+    idx = [model_col] + [c for c in rfc_col if c in df.columns]
+
+    tyndp_str = "TYNDP 2024"
+    if "TYNDP 2024 Vis Plfm" in idx and tyndp_str in idx:
```

Comment:
Only if my peeve from above is accepted 😄
```suggestion
    if "TYNDP 2024 Vis Pltfm" in idx and tyndp_str in idx:
```